--- mixxx-2.5.2/CMakeLists.txt.orig	2025-08-29 06:22:49.195412802 +0200
+++ mixxx-2.5.2/CMakeLists.txt	2025-08-29 06:23:33.843291598 +0200
@@ -1813,7 +1813,7 @@ else()
 endif()
 
 set_target_properties(mixxx-lib PROPERTIES CXX_CLANG_TIDY "${CLANG_TIDY}")
-target_link_libraries(mixxx PRIVATE mixxx-lib mixxx-gitinfostore)
+target_link_libraries(mixxx PRIVATE mixxx-lib mixxx-gitinfostore Microsoft.GSL::GSL)
 
 #
 # Installation and Packaging
@@ -2232,7 +2232,7 @@ if(BUILD_TESTING)
 
   target_precompile_headers(mixxx-test REUSE_FROM mixxx-lib)
   set_target_properties(mixxx-test PROPERTIES AUTOMOC ON)
-  target_link_libraries(mixxx-test PRIVATE mixxx-lib mixxx-gitinfostore GTest::gtest GTest::gmock)
+  target_link_libraries(mixxx-test PRIVATE mixxx-lib mixxx-gitinfostore GTest::gtest GTest::gmock Microsoft.GSL::GSL)
 
   if(BUILD_BENCH)
     add_compile_definitions(USE_BENCH)
