--- mixxx-2.5.2/CMakeLists.txt.orig	2025-08-29 06:18:22.546245677 +0200
+++ mixxx-2.5.2/CMakeLists.txt	2025-08-29 06:19:12.115896755 +0200
@@ -2423,9 +2423,9 @@ if(ENGINEPRIME)
   # statically.  This situation should be reviewed once libdjinterop hits version 1.x.
   set(LIBDJINTEROP_VERSION 0.24.3)
   # Look whether an existing installation of libdjinterop matches the required version.
-  find_package(DjInterop  ${LIBDJINTEROP_VERSION} EXACT CONFIG)
+  find_package(DjInterop  ${LIBDJINTEROP_VERSION} CONFIG)
   if(NOT DjInterop_FOUND)
-    find_package(DjInterop  ${LIBDJINTEROP_VERSION} EXACT MODULE)
+    find_package(DjInterop  ${LIBDJINTEROP_VERSION} MODULE)
   endif()
 
   if(DjInterop_FOUND)
