--- mixxx-2.4.1/CMakeLists.txt.orig	2024-09-28 21:47:06.674846532 +0200
+++ mixxx-2.4.1/CMakeLists.txt	2024-09-28 22:01:31.953175308 +0200
@@ -2283,7 +2283,7 @@ if(ENGINEPRIME)
   # statically.  This situation should be reviewed once libdjinterop hits version 1.x.
   set(LIBDJINTEROP_VERSION 0.20.2)
   # Look whether an existing installation of libdjinterop matches the required version.
-  find_package(DjInterop  ${LIBDJINTEROP_VERSION} EXACT CONFIG)
+  find_package(DjInterop  ${LIBDJINTEROP_VERSION} CONFIG)
   if(NOT DjInterop_FOUND)
     find_package(DjInterop  ${LIBDJINTEROP_VERSION} EXACT MODULE)
   endif()
